src/components/Button/Button.jsx

```diff
@@ -0,0 +1,29 @@
+import PropTypes from "prop-types";
+import "./Button.scss";
+
+const Button = ({ children, className, onClickHandler, type }) => {
+  return (
+    <button
+      type={type}
+      className={`${className} custom-button`}
+      onClick={onClickHandler}
+    >
+      {children}
+    </button>
+  );
+};
+
+Button.defaultProps = {
+  className: null,
+  onClickHandler: null,
```

Comment:
```suggestion
  onClickHandler: () => {},
```
please follow this everywhere applicable.

Reply:
Okay @maa-wtag bhai. Followed.